pages/rooms/create/mood/index.tsx

```diff
@@ -50,8 +83,8 @@ const RoomCreateMoodPage: NextPage = () => {
           {MOOD_EXAMPLE.map((v) => (
             <StyledButton
               key={v.text}
-              onClick={() => setMood(v.iconName)}
-              isActive={mood === v.iconName}
+              onClick={() => setMood({ name: v.text, emoji: v.iconName })}
+              isActive={mood.emoji === v.iconName}
```

Comment:
```suggestion
          {MOOD_EXAMPLE.map((mood) => (
            <StyledButton
              key={mood.text}
              onClick={() => setMood({ name: mood.text, emoji: mood.iconName })}
              isActive={mood.emoji === mood.iconName}
```

같은 네이밍으로 mood인데 내용이 달라서 더 좋은 네이밍이 있다면 좋겠다
```typescript
   {
      name: string;
      emoji: string;
    }
```
```typescript
   {
      name: string;
      iconName: string;
    }
```

Reply:
@manudeli [298ca4a](https://github.com/mash-up-kr/musily-web/pull/85/commits/298ca4a481030fec3176503c4d8a291901cb6d8e) 에서 수정했어요~